### ISSUE TITLE
Page Edit/Review: About Contributing

### ISSUE THREAD

**Issue by buttonwillowsix** [SU-SWS/decanter#564]
https://elegant-poitras-87214a.netlify.com/page/about-contributing/

Should this be split into more than one page?  Also needs a Designer section.

**Comment by buttonwillowsix** [comment]
I think this page is ready for review. The two issues I see:

-Is that place where it indicates using grunt still correct?
-Do we need to modify the language around the license and open source? (Or modify the license.)

**Comment by sherakama** [comment]
Yeah no more grunt and we use the MIT license for the code. The terms for the use of the brand is different than the code so I think it is all acceptable.

**Comment by sherakama** [comment]
I removed the grunt stuff.
